src/test/java/com/google/step/finscholar/firebase/FirebaseStorageManagerTest.java

```diff
@@ -33,13 +40,31 @@ public void isRecentlySetDocumentRetrievable() {
     } catch (Exception e) {
       System.out.println(e.toString());
     }
-    String json = "";
+    String json = ServletConstantValues.DEFAULT_VALUE;
     try {
       json = FirebaseStorageManager.getDocument(firebase, ServletConstantValues.TEST_COLLECTION_NAME, ServletConstantValues.TEST_DOCUMENT_NAME);
     } catch (Exception e) {
       System.out.println(e.toString());
     }
-    Assert.assertEquals(json, ServletConstantValues.EXPECTED_DOCUMENT_RETRIEVABLE);
+    Assert.assertEquals(ServletConstantValues.EXPECTED_DOCUMENT_RETRIEVABLE, json);
+  }
+
+
+  @Test
+  public void setMultipleDocumentsAndRetrieveCollection() {
+    try {
+      FirebaseStorageManager.storeMultipleDocuments(firebase, ServletConstantValues.TEST_COLLECTION_NAME, testObjectList);
+    } catch (Exception e) {
+      System.out.println(e.toString());
+    }
+    String json = ServletConstantValues.DEFAULT_VALUE;
+    try {
+      json = FirebaseStorageManager.getCollection(firebase, ServletConstantValues.TEST_COLLECTION_NAME);
+    } catch (Exception e) {
+      System.out.println(e.toString());
+    }
+    System.out.println(json);
```

Comment:
Do you need this line?

Reply:
Not really... I was mostly using it for debugging